### DISCUSSION TITLE
How to provide output.txt to gpt-4

### DISCUSSION THREAD

**Discussion by yordanboykov-ibm** [mpoon/gpt-repository-loader#31] [Q&A] [answered]
Hi,
could you please provide example how can we provide output.txt to gpt and how PR works?

**Comment by zackees** [comment, marked as answer]
You have to have the pro version of chat gpt. The smaller ones just can't accept a lot of text.

You simply go here:

https://chat.openai.com/chat?model=gpt-4

Then paste in the text.

**Comment by ryskin** [comment]
but how would a repository with hundred files be copy-pasted to GPT4 chat input?

**Reply by zackees** [reply]
output.txt contains the concatenated files in the repo with special AI commands.

You simply copy and paste the output.txt file to chat gpt then ask your question

**Reply by zackees** [reply]
Unless your repo is incredibly small, you should be using ChatGPT4-32k, which allows up to ~52 pages of text as input
